src/node_options.cc

```diff
@@ -24,6 +24,44 @@ enum {
 #endif
 };
 
+// Normalize the argument vector by exploding
+// multiple options (if any). For example
+// "node -abc --scm git" -> "node -a -b -c --scm git"
+// implementation was adapted from
+// https://github.com/clibs/commander/blob/master/src/commander.c
+static char ** normalize_args(int *argc, char **argv) {
+  int size = 0;
+  int alloc = *argc + 1;
+  char **nargv = static_cast<char **>(malloc(alloc * sizeof(char *)));
+  int i;
+
+  for (i = 0; argv[i]; ++i) {
```

Comment:
Oh, and you should probably start counting from one?  `argv[0]` is the program name.